src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,18 @@
 package core.basesyntax;
 
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+    private static final int COLORS_SIZE = 7;
```

Comment:
why do we need this? we can use COLORS.length

Reply:
Done